concepticondata/conceptlists.tsv

```diff
@@ -240,3 +240,4 @@ Carling-2017-87k	Carling, Gerd and Larsson, Filip and Cronhamn, Sandra and Farre
 Nagano-2013-1256	Nagano, Yasuhiko and Prins, Marielle	2013		1256	questionnaire,areal	English	South-East Asian Languages	https://htq.minpaku.ac.jp/databases/rGyalrong/lang/area/index.php?langindex=eng	Nagano2013		The results of an international joint survey of the rGyalrongic languages, originally published by the authors in their own website.		
 Yang-2011-1014	Yang, Cathryn	2011		1014	questionnaire	Chinese	Lalo Languages	https://www.sil.org/system/files/reapdata/45/63/47/45634767234331504727329755903316120678/Yang_Lalo_Sept_2011.pdf	Yang2011		A survey of Lalo regional varieties for research in phylogenetics, dialectometry and sociolinguistics.		
 Vejdemo-2016-167	Vejdemo, Susanne and Hörberg, Thomas	2016		167	basic, ranked	English	Global	https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0147924#pone.0147924.ref032 	Vejdemo2016		This list represents 167 content words, built upon the list by [Pagel et al. (2007)](:ref:Pagel-2007-200). It was originally compiled with the purpose of assessing the rate of lexical replacement for the specified concepts in English, Danish, Dutch, German and Swedish, according to word class, frequency, age of acquisition, synonyms, arousal, imageability and average mutual information either from published databases or from corpora and lexica.		
+Lieberherr-2017-100	Lieberherr, Ismael and Bodt, Timotheus Adrianus	2017		100	basic, areal	English	Kho-Bwa languages	https://escholarship.org/uc/item/4t27h5fg	Lieberherr2017		The given list was compiled to test the relatedness of 22 Kho-Bwa languages, supposedly constituting a separate branch within the Tibeto-Burman subgroup. The analysis of their shared core vocabulary also included 7 other Tibeto-Burman varieties to contrast them with the Kho-Bwa languages. This list is mainly based on the Leipzig-Jakarta 100 concept list (Haspelmath and Tadmor 2009), with some entries, considered ambiguous by the compilers, being replaced with items from [the Swadesh list (1971)](:ref:Swadesh-1971-100).		
```

Comment:
this is not an areal list, as it does not occur to be in opposition to normal swadesh.